### PR TITLE
Write test for post API controller

### PR DESCRIPTION
**Pull request by ibramoha**

### CODE REVIEW

File: backend/test/api_spec.js [hunk 1]
```diff
@@ -4,115 +4,73 @@ import { expect } from 'chai';
...
-    const postcode = { postcod: 'W7 3DR' };
+  it('should not save unless correct data has been supplied', async () => {
+    const postcode = { lat: 'nw78', long: '657858' };
     const result = await request(app)
       .post('/api/service/postcode')
-      .send({ postcode });
+      .send(postcode);
     expect(result.statusCode).to.be.equal(200);
-    expect(result.body.success).to.be.true;
+    expect(result.body.success).to.be.equal(true)
```
Comment (DwinaTech) [inline backend/test/api_spec.js]: change this to expect(result.body.success).to.equal(true)

File: backend/test/api_spec.js [hunk 2]
```diff
@@ -121,7 +79,7 @@ describe('API testing', async () => {
       .post('/api/service/organisation/add')
       .send(anOrg);
     expect(result.statusCode).to.be.equal(200);
-    expect(result.body.success).to.be.true;
+    expect(result.body.success).to.be.equal(true);
```
Comment (DwinaTech) [inline backend/test/api_spec.js]: ditto

Comment (ibramoha) [reply backend/test/api_spec.js]: Done

File: backend/test/api_spec.js (hunk 1 above)
Comment (ibramoha) [reply backend/test/api_spec.js]: Done

**Pull request merged**